### DISCUSSION TITLE
Reset Audio Features settings

### DISCUSSION THREAD

**Discussion by antscribano** [Marekkon5/onetagger#255] [Q&A] [answered]
Hello,
I accidentally changed the Audio Features Properties mapping of some audio features:

![image](https://github.com/Marekkon5/onetagger/assets/8087993/bcc8b9a3-4621-4927-949c-cf11ae1d71b7)

**Is there a way to set it to their default values (i.e. 1T_DANCEABILITY, 1T_ENERGY...)?** so I can just use the global settings?

Thank you

**Comment by Marekkon5** [comment, marked as answer]
Hello, there is currently no way to restore the default settings, however you can just type back the `1T_DANCEABILITY` and `1T_ENERGY`.
